src/NzbDrone.Core/Notifications/CustomScript/CustomScript.cs

```diff
@@ -127,6 +127,56 @@ public override void OnRename(Series series)
             ExecuteScript(environmentVariables);
         }
 
+        public override void OnDelete(EpisodeDeleteMessage deleteMessage)
+        {
+            var series = deleteMessage.Series;
+            var episodeFile = deleteMessage.EpisodeFile;
+
+            var environmentVariables = new StringDictionary();
+
+            environmentVariables.Add("Sonarr_EventType", "Episode Deleted");
```

Comment:
This is more of an enum, not a description string, so it should be `EpisodeDeleted`

Reply:
Resolved in upcoming commit